src/SFA.DAS.Campaign.Web/Controllers/EmployerController.cs

```diff
@@ -1,23 +1,106 @@
-﻿using Microsoft.AspNetCore.Mvc;
+﻿using MediatR;
+using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Options;
+using SFA.DAS.Campaign.Application.Content.Queries;
+using SFA.DAS.Campaign.Application.FundingTool;
+using SFA.DAS.Campaign.Domain.ApprenticeshipCourses;
+using SFA.DAS.Campaign.Infrastructure.Api.Responses;
 using SFA.DAS.Campaign.Infrastructure.Configuration;
+using SFA.DAS.Campaign.Web.Helpers;
+using SFA.DAS.Campaign.Web.Models;
+using System.Threading.Tasks;
+using Standard = SFA.DAS.Campaign.Application.FundingTool.Standard;
 
 namespace SFA.DAS.Campaign.Web.Controllers
 {
     public class EmployerController : Controller
     {
         private readonly CampaignConfiguration _configuration;
+        private readonly IMediator _mediator;
+        private readonly IStandardsRepository _repository;
 
-        public EmployerController (IOptions<CampaignConfiguration> configuration)
+        private const string calculationPanel1Slug = "are-you-ready-to-get-going";
+        private const string calculationPanel2Slug = "future-proof-your-business";
+
+        public EmployerController(IOptions<CampaignConfiguration> configuration, IMediator mediator, IStandardsRepository repository)
         {
             _configuration = configuration.Value;
+            _mediator = mediator;
+            _repository = repository;
         }
-        
+
         [Route("/employers/find-apprenticeship-training")]
         [Route("/employer/find-apprenticeship-training")]
         public IActionResult FindApprenticeshipTraining()
         {
             return RedirectPermanent(_configuration.FatBaseUrl);
         }
+
+        [Route("/employers/calculate-your-apprenticeship-funding")]
+        public async Task<IActionResult> CalculateApprenticeshipFunding([FromQuery] bool preview = true)
+        {
+            var standards = _repository.GetStandards();
```

Comment:
As discussed. I think it's worth changing the two GetStandards and GetStandardCalls to be use mediator ( I think a sector one may already exist ) which would reduce the injection of the repository into the controller directly.